uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/ContactTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use CRM_HRCore_Test_Fabricator_Contact as ContactFabricator;
+
+
+/**
+ * Class api_v3_LeaveRequestTest
+ *
+ * @group headless
+ */
+class api_v3_ContactTest extends BaseHeadlessTest {
+
+  use CRM_HRLeaveAndAbsences_LeaveRequestHelpersTrait;
+  use CRM_HRLeaveAndAbsences_LeaveBalanceChangeHelpersTrait;
+  use CRM_HRLeaveAndAbsences_LeaveManagerHelpersTrait;
+  use CRM_HRLeaveAndAbsences_SessionHelpersTrait;
+
+  private $manager;
+
+  private $contact1;
+
+  private $contact2;
+
+  public function setUp() {
+    CRM_Core_DAO::executeQuery('SET foreign_key_checks = 0;');
+
+    $this->contact1 = ContactFabricator::fabricate();
+    $this->contact2 = ContactFabricator::fabricate();
+    $this->manager = ContactFabricator::fabricate();
+    $this->registerCurrentLoggedInContactInSession($this->manager['id']);
+
+    $this->setLeaveApproverRelationshipTypes([
+      'approves leaves for',
+      'manages things for',
```

Comment:
These relationship names are confusing. Basically, it says CONTACT A approves leaves for CONTACT B, i.e. the leave approver is the contact A, but on ContactSelect, the ID of the current logged user is checked against contact B.

I've checked the existing code and I see we're using these relationship names in a few other places (git blame says it's my fault. Sorry!). Can you please fix this here and in these other places?